### PR TITLE
feat: Multisig import and export config compatible with ckb-cli

### PR DESCRIPTION
**Pull request by yanguoyu**
Multisig config created by ckb-cli can also import by Neuron.

https://user-images.githubusercontent.com/12881040/177503032-6392fb72-4881-49d9-8e64-2043efa990d9.mov

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/multisig.ts [hunk 1]
```diff
@@ -104,27 +115,28 @@ export default class MultisigController {
...
       if (
-        configs.some(config => config.r === undefined
-          || config.m === undefined
-          || config.n === undefined
-          || config.blake160s === undefined
-          || config.r > config.n
-          || config.m > config.n
-          || !config.blake160s.length
-          || config.blake160s.some(v => v.length !== SPEC256_BLAKE160_LEN)
+        !configOutput.multisig_configs ||
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/multisig.ts:120]: It's kinda complex, maybe we could add a specific function and add test cases to show its rule.

File: packages/neuron-wallet/src/controllers/multisig.ts [hunk 2]
```diff
@@ -2,15 +2,40 @@ import fs from 'fs'
...
+  return configOutput.multisig_configs &&
+  Object.values(configOutput.multisig_configs).length &&
+  Object.values(configOutput.multisig_configs).every(config => config.require_first_n !== undefined
+    && config.threshold !== undefined
+    && config.sighash_addresses !== undefined
+    && !Number.isNaN(+config.require_first_n)
+    && !Number.isNaN(+config.threshold)
+    && config.sighash_addresses?.length
+    && config.sighash_addresses?.length >= +config.require_first_n
+    && config.sighash_addresses?.length >= +config.threshold
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/multisig.ts:35]: These 7 statements could be done in `config.sighash_address?.length >= Math.max(+config.require_first_n, +config.threshold)`

File: packages/neuron-wallet/src/controllers/multisig.ts [hunk 3]
```diff
@@ -2,15 +2,40 @@ import fs from 'fs'
...
+interface MultisigConfigOutput {
+  multisig_configs: Record<string, {
+    sighash_addresses: string[],
+    require_first_n: number,
+    threshold: number
+    alias?: string
+  }>
+}
+
+const validImportConfig = (configOutput: MultisigConfigOutput) => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/multisig.ts:25]: `validImportConfig` => `validateImportConfig` or `isImportConfigValid`

Comment (Keith-CY) [reply packages/neuron-wallet/src/controllers/multisig.ts:25]: Add tests on this method

Comment (yanguoyu) [reply packages/neuron-wallet/src/controllers/multisig.ts:25]: I have tested it by `importConfig` function.

**Review by Keith-CY — APPROVED** [review]

**Review by Cedar67 — APPROVED** [review]
&#x2705; Import the multisig address file generated by the CKB CLI for the first time.
&#x2705; Import the multisig address file generated by the CKB CLI repeatedly.
&#x2705; Import a [1 of 2] multisig address file generated by the CKB CLI.
&#x2705; Import a [2 of 2]  multisig address file generated by the CKB CLI.
&#x2705; View the multisig address file generated by CKB CLI in Neuron.
&#x2705; Export the multisig address file generated by the CKB CLI.
&#x2705; Transation is performed by Neuron through the multisig address generated by CKB CLI.
&#x2705; Approvement is performed by Neuron through the multisig address generated by CKB CLI.

**Pull request merged**